backend/python/transformers-musicgen/test.sh

```diff
@@ -3,9 +3,9 @@
 ## A bash script wrapper that runs the transformers server with conda
 
 # Activate conda environment
-source conda activate transformers-musicgen
```

Comment:
this was crashing hardly locally. Wondering @dave-gray101 if that worked for you?